i18n.php

```diff
@@ -59,6 +59,21 @@
     'then' => 'اذاً|ثم',
     'when' => 'عندما|متى',
   ),
+  'az' => 
+  array (
+    'and' => 'Və/Həm',
```

Comment:
My bad. It is not allowed I know. Didn't check. By the way I sent PR to cucumber/gherkin and they merged it.